src/common/statbar.cpp

```diff
@@ -226,7 +226,7 @@ wxArrayInt wxStatusBarBase::CalculateAbsWidths(wxCoord widthTotal) const
                 widths.Add(pane.GetWidth());
             else
             {
-                int nVarWidth = widthExtra > 0 ? (widthExtra * (-pane.GetWidth())) / nVarCount : 0;
+                int nVarWidth = (widthExtra > 0 && nVarCount != 0) ? (widthExtra * (-pane.GetWidth())) / nVarCount : 0;
```

Comment:
Sorry, this is not unlikely, this is impossible because the loop above makes sure `nVarCount` is non-zero in this case. I also dislike warnings, but this makes the code harder to understand, so I don't like this fix either.

Can we use something like `__builtin_assume(nVarCount > 0)` (wrapped in a macro, as usual) instead?

Reply:
I'm OK with doing nothing. Macro decorations will make it harder to read than my original change.